eng/Directory.Build.Data.targets

```diff
@@ -33,12 +35,20 @@
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-alpha-004" PrivateAssets="all" IsImplicitlyDefined="true" />
   </ItemGroup>
 
-  <PropertyGroup Condition="'$(SignAssembly)' != 'false'">
+  <!-- If signing is enabled for an SDK library, delay sign using the shared key -->
```

Comment:
With my suggestions below then you should be able to eliminate the condition changes here.